prismic/custom-types/events.json

```diff
@@ -208,6 +208,30 @@
           }
         }
       }
+    },
+    "events" : {
+      "type" : "Group",
+      "fieldset" : "Event Partners",
+      "config" : {
+        "fields" : {
+          "data" : {
+            "type" : "Link",
+            "config" : {
+              "select" : "document",
+              "customtypes" : [ "partner" ],
+              "placeholder" : "Link to a partner"
+            }
+          },
+          "level" : {
```

Comment:
That's because it only exists on the association between the partner and an event.